.buildkite/publish/docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       - TEST_MONGO_URI=mongodb://root:prisma@mongo:27018/tests?authSource=admin
       - TEST_MONGO_URI_MIGRATE=mongodb://root:prisma@mongodb_migrate:27017/tests-migrate?authSource=admin
       # CockroachDB
-      - TEST_COCKROACH_URI=postgresql://prisma@cockroachdb:26257/
+      - TEST_COCKROACH_URI=postgresql://prisma@cockroachdb:26257/tests
```

Comment:
Is this env var already used anywhere in code? 
(If thought there were not CockroachDB tests - why do we have the env var then?)

Reply:
This PR is the first that introduces cockroachdb tests